projects/plugins/crm/tests/codeception/acceptance.suite.dist.yml

```diff
@@ -8,8 +8,8 @@ actor: AcceptanceTester
 modules:
     enabled:
         - Db:
-            user: 'some_db_user'
-            password: 'some_db_pass'
+            user: 'gogdzl'
+            password: 'pass'
```

Comment:
This looks like a leftover from local development 😄 
AFAIK then you can create other named files instead of "dist" (`acceptance.suite.dist.yml`) for your local environment which hopefully will make this easier in the future.

We currently ignore `/tests/codeception/acceptance.suite.yml` in `.gitignore` so that might be a solution? 😄

Reply:
I was having some problems running my tests, so one of the things I tried to do to help me debug it was to rename my `acceptance.suite.yml` to `dist`. Once I solved my problem I forgot to revert that :(

Reply:
No worries! That's why we have reviews 😄

Reply:
... and now everyone has access to your super secret password 😄